### PR TITLE
Store Error Verbosity in Chrome Storage

### PR DESCRIPTION
**Pull request by Bracciata**

### CODE REVIEW

**Comment by Bracciata** [comment]
Closes #97

File: extension/index.html [hunk 1]
```diff
@@ -1,85 +1,94 @@
...
+      <b>tailored directly around the users</b> disability. When users first
+      signup, they can select their disability and see personally-tailored
+      results; otherwise, there is a default score for
+      <b>general accessibility</b>.
+    </p>
+
+    <h3>
+      Note that in the form below your changes will save automatically:
+    </h3>
+    <h2>Settings:</h2>
```
Comment (emma-hubka) [inline extension/index.html:35]: maybe remove the colon here because there is already one on the line before

File: extension/index.html [hunk 2]
```diff
@@ -1,85 +1,94 @@
...
+    <form>
+      <p>What do you identify with?</p>
```
Comment (emma-hubka) [inline extension/index.html]: can you make the questions a bit bigger?

Comment (emma-hubka) [reply extension/index.html]: there is also more space after the question than before and that makes the grouping a bit odd

**Review by emma-hubka — CHANGES_REQUESTED** [review]

File: extension/index.html [hunk 3]
```diff
@@ -1,85 +1,95 @@
...
+      directly in a google search
+      <b>tailored directly around the users</b> disability. When users first
+      signup, they can select their disability and see personally-tailored
+      results; otherwise, there is a default score for
+      <b>general accessibility</b>.
+    </p>
+
+
+    <h2>Settings:</h2>
+     <h3>
```
Comment (emma-hubka) [inline extension/index.html]: the note is very large now

File: extension/index.html [hunk 4]
```diff
@@ -1,85 +1,95 @@
...
+        <input type="checkbox" id="motor" name="motor" />
+        Motor Function Disability </label><br />
+
+      <label for="sightedkeyboarduser">
+        <input type="checkbox" id="sightedkeyboarduser" name="sightedkeyboarduser" />
+        Sighted Keyboard User
+      </label>
+      <br />
+
+      <h4>
```
Comment (emma-hubka) [inline extension/index.html]: it would be nice for there to be some space before this question

File: extension/index.html [hunk 5]
```diff
@@ -1,85 +1,95 @@
...
+        How descriptive would you like error messages?
```
Comment (emma-hubka) [inline extension/index.html:78]: I like the size and format of the questions now

**Review by emma-hubka — CHANGES_REQUESTED** [review]

**Pull request merged**